### DISCUSSION TITLE
Github Education won't authenticate me

### DISCUSSION THREAD

**Discussion by yuiveslkao** [community/community#121669] [GitHub Education]
### Hi everyone,

Help.
I am a student at a Japanese university and I wanted to sign up for Github education, but Github never authenticated me, no matter how many times I uploaded my student ID. What should I do?

**Comment by theri6v** [comment]
That sounds frustrating! I also Faced the same situation once I followed below things and it get shorted :

1)Ensure that you're meeting all the requirements for student verification outlined by GitHub.
2) If you're having trouble with the verification process, reach out to GitHub Support directly.
3)If you're having trouble with your student ID, see if there are alternative forms of documentation you can provide to verify your student status.
